frontend/src/app/libs/event-description/components/event-description-panel/event-description-panel.component.html

```diff
@@ -1,27 +1,92 @@
 <div class="app-event-description-panel">
-  <div class="app-event-description-panel__title">
-    <div>{{event.title}}</div>
-  </div>
-  <ng-container *ngIf="event.endDate else withoutEndDateTemplate">
-    <div class="app-event-description-panel__date">
-      {{ event.startDate | date:'d MMM yy' }} - {{ event.endDate | date:'d MMM yy' }}
-    </div>
-    <div class="app-event-description-panel__time">
-      {{ event.startTime | date:'h a' }} - {{ event.endTime | date:'h a' }}
-    </div>
-  </ng-container>
-  <ng-template #withoutEndDateTemplate>
-    <div class="app-event-description-panel__date">
-      {{ event.startDate | date:'d MMM yy' }}
-    </div>
-    <div class="app-event-description-panel__time">
-      <span>In Progress</span>
-    </div>
-  </ng-template>
-  <div class="app-event-description-panel__description" [innerHTML]="event.description"></div>
-  <div class="app-event-description-panel__tool-tip" matTooltip="You must be logged in to participate in this round"
-    [matTooltipDisabled]="userLoginStatus">
-    <button mat-raised-button class="app-event-description-panel__take-part-button" [disabled]="!userLoginStatus">TAKE
-      PART</button>
-  </div>
+	<div class="app-event-description-panel__breadcrumb">
+		<app-breadcrumb [event]="event"></app-breadcrumb>
+	</div>
+
+	<div class="app-event-description-panel__container {{event?.color}}">
+		<div class="app-event-description-panel__box-left">
+			<div class="app-event-description-panel__title">{{event.title}}</div>
+			<div class="app-event-description-panel__description" [innerHTML]="event.description"></div>
```

Comment:
Please, use sanitizer to provide save value for innerHTML.
Here you can found examples:
https://www.intricatecloud.io/2019/10/using-angular-innerhtml-to-display-user-generated-content-without-sacrificing-security/#:~:text=Angular%20comes%20with%20a%20built,getting%20removed%20from%20your%20page.

Reply:
https://stackoverflow.com/questions/45140911/angular-domsanitizer-security-bypass-innerhtml-styling-weird-reloading-effec
This is a good example.